lib/generators/templates/models/remote_account.rb

```diff
@@ -0,0 +1,6 @@
+class RemoteAccount < BookingsyncPortal::RemoteAccount
+  # Returns the name of this portal account. Should be easy to identify.
+  def name
```

Comment:
We use it on the view: https://github.com/BookingSync/bookingsync_portal/blob/master/app/views/bookingsync_portal/admin/rentals/index.html.erb#L27